nntrainer/tensor/tensor_pool.cpp

```diff
@@ -259,6 +259,23 @@ TensorPool::requestSpec &TensorPool::getSourceSpec(const std::string &name) {
   return pool.at(parent_spec_idx);
 }
 
+Tensor *TensorPool::placeholder(const std::string &name, const TensorDim &dim) {
+  /// @todo rename requestExternallyAllocateTensor -> placeholder
+  return requestExternallyAllocateTensor(dim, name);
+}
+
+Tensor *TensorPool::create(const std::string &name, const TensorDim &dim,
+                           const std::vector<unsigned int> &exec_order,
+                           TensorLifespan lifespan,
+                           const Tensor::Initializer &init) {
+  /// @todo rename requestTensor -> create
+  return requestTensor(dim, exec_order, lifespan, name, init);
+}
+
+bool TensorPool::tensorExist(const std::string &name) {
+  return name_map.count(name);
```

Comment:
consider using `getTensor(name) != nullptr`.

Reply:
Updated as suggested

Reply:
getTensor's behavior does not return nullptr instead throw, so reverted to the current.

Reply:
Consider using getTensor() later (maybe just add a TODO for now) which makes it simple to update internals of TensorPool if needed.

Reply:
I'll put todo here.